tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -486,6 +486,21 @@ _ when x.ToBoolean() is { } boolean => boolean,
             telemetry.Record(ConfigTelemetryData.SsiInjectionEnabled, value: EnvironmentHelpers.GetEnvironmentVariable("DD_INJECTION_ENABLED"), recordValue: true, ConfigurationOrigins.EnvVars);
             telemetry.Record(ConfigTelemetryData.SsiAllowUnsupportedRuntimesEnabled, value: EnvironmentHelpers.GetEnvironmentVariable("DD_INJECT_FORCE"), recordValue: true, ConfigurationOrigins.EnvVars);
 
+            if (EnvironmentHelpers.GetEnvironmentVariable("DD_INJECTION_ENABLED")?.Contains("tracer") == true)
+            {
+                telemetry.Record(ConfigTelemetryData.InstrumentationSource, "ssi", recordValue: true, ConfigurationOrigins.EnvVars);
+            }
+            else
+            {
+                string? value = EnvironmentHelpers.GetEnvironmentVariable("DD_INSTRUMENTATION_INSTALL_TYPE");
```

Comment:
FYI, this value is already tracked separate by telemetry - it's explicitly part of the `app-started` payload 🤔

Reply:
Yup, I changed this a little bit. The accepted values for `instrumentation_source` will be `ssi`, `cmd_line` (which should detect if ddtool was used), and `env_var`.